src/Areas/Storage/StorageSetup.cs

```diff
@@ -63,5 +63,7 @@ public void RegisterCommands(CommandGroup rootGroup, ILoggerFactory loggerFactor
 
         fileSystem.AddCommand("list-paths", new FileSystemListPathsCommand(
             loggerFactory.CreateLogger<FileSystemListPathsCommand>()));
+        fileSystem.AddCommand("directory-create", new DirectoryCreateCommand(
```

Comment:
This is not under fileSystem group.

It is a new group directory and the operation is create.

It used DataLakeDirectoryClient.